membership/interface.go

```diff
@@ -0,0 +1,14 @@
+package membership
+
+// Member defines a member of the membership that can be used by the rest of the
+// system
+type Member interface {
+	// GetAddress returns the external address used by the rpc layer to
+	// communicate to the member. It is prefixed with Get for legacy reasons and
+	// can be removed after a refactor of the swim.Member
```

Comment:
Can't we do that before we extract and interface? Otherwise we won't be able to get rid of it without (another) breaking change.